src/spetlr/autoloader/autoloader_handle.py

```diff
@@ -0,0 +1,62 @@
+from pyspark.sql import DataFrame
+
+from spetlr.configurator.configurator import Configurator
+from spetlr.delta.delta_handle import DeltaHandleInvalidFormat
+from spetlr.exceptions import SparkVersionNotSupportedForSpetlrStreaming
+from spetlr.spark import Spark
+from spetlr.tables import TableHandle
+
+
+class AutoloaderHandle(TableHandle):
+    def __init__(
+        self,
+        *,
+        location: str,
+        schema_location: str,
+        data_format: str,
+    ):
+        """
+        location: the location of the delta table
+        checkpoint_path: The location of the checkpoints, <table_name>/_checkpoints
```

Comment:
I see now that it is only this docstring that needs updating since you changed the parameter name to be the same as the databricks argument. I'm going to close the other comments.